tests/test_collection.py

```diff
@@ -413,6 +414,46 @@ def test_to_from_dict(self) -> None:
 
         self.assertDictEqual(extent_dict, extent.to_dict())
 
+    def test_utc_datetimes_are_not_localized(self) -> None:
```

Comment:
Same comment as in `test_item.py`, these branch checks could be moved to `test_utils.py` and then this could just check the basic regression case.

Reply:
Fixed in 6997000e40f248ea8779f53d5b41c3f535aa9a51